devchat/openai/openai_chat.py

```diff
@@ -34,14 +35,15 @@ class OpenAIChat(Chat):
     """
     OpenAIChat class that handles communication with the OpenAI Chat API.
     """
-    def __init__(self, config: OpenAIChatConfig):
+    def __init__(self, config: Any, extra_config: Any = None):
         """
```

Comment:
This is ugly. The current abstraction should be good (let me know if it makes any fundamental trouble). So, it is better to add a parallel implementation like OpenAIXXX classes, for general models, instead of extending OpenAIChat this way.